android/app/src/main/java/com/mapeo/MainActivity.java

```diff
@@ -10,11 +10,11 @@
 import com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView;
 /* End: react-native-guesture-handler */
 
-public class MainActivity extends ReactFragmentActivity {
+public class MainActivity extends ReactActivity {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         SplashScreen.show(this);  // react-native-splash-screen
-        super.onCreate(savedInstanceState);
+        super.onCreate(null); // react-native-screens
```

Comment:
this is from the `react-native-screens` [docs](https://github.com/software-mansion/react-native-screens#android) so generally an uninformed change on my part 😅 can try reverting that change and seeing if anything breaks, since I didn't specifically test for it. Original issue is [here](https://github.com/software-mansion/react-native-screens/issues/17)

Reply:
the same change is also mentioned in the current version of the react-navigation docs https://reactnavigation.org/docs/getting-started/#installing-dependencies-into-a-bare-react-native-project

Reply:
Great, I'm going to assume that this is the way to do it, since it's also referenced in the older version of `expo-splash-screen` (side-note, perhaps we should switch to expo-splash-screen in the future, since it should be better maintained).